packages/url_launcher/url_launcher_platform_interface/lib/url_launcher_platform_interface.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+/// The interface that implementations of url_launcher must implement.
+abstract class UrlLauncherPlatform {
```

Comment:
The static platform field should be on this class and not on the app-facing plugin (so that platform implementations don't need to depend on the app-facing package)

Reply:
Done.